add-on/panels/js/panel.jsx

```diff
@@ -128,16 +128,16 @@ loop.panel = (function(_, mozL10n) {
       var locale = mozL10n.getLanguage();
       var tosHTML = mozL10n.get("legal_text_and_links3", {
         "clientShortname": mozL10n.get("clientShortname2"),
-        "terms_of_use": React.renderToStaticMarkup(
+        "terms_of_use": "<a>Fake</a>", /* ReactDOM.render(
```

Comment:
I've had a chat to the L10n folks, and it seems like our best option may be to upgrade our l10n-*.js as then we'd able to use constructs such as:

    legal_text="You agree to the <a>terms_of_use</a>"

in the L10n file, and be able to specify attributes for the element. I'm not 100% sure it'll work properly for two `<a>` replacements in the same string, but we'll have to take that as it comes.

The bug for this is https://bugzilla.mozilla.org/show_bug.cgi?id=1230208 - and I think we'll have to block this bug on at least investigating that. It feels like the upgrade of L10n.js is going to need some testing and investigation so we're going to need to schedule that in.